extensions/wikia/TemplateDraft/controllers/TemplateDraftController.class.php

```diff
@@ -59,4 +59,43 @@ public function approveDraft( Title $title ) {
 		// Save to parent page
 		$page->doEdit( $draftContent, wfMessage( 'templatedraft-approval-summary' )->inContentLanguage()->plain() );
 	}
+
+	public function markTemplateAsNotInfobox() {
+		/**
+		 * First, validate a request
+		 */
+		if ( !$this->wg->User->isPowerUser()
+			|| !$this->isValidPostRequest()
+		) {
+			$this->response->setVal( 'status', false );
+			return false;
+		}
+
+		/**
+		 * Then check the pageId param
+		 */
+		$pageId = $this->getRequest()->getInt( 'pageId' );
+		if ( $pageId === 0 ) {
+			$this->response->setVal( 'status', false );
+			return false;
+		}
+
+		/**
+		 * Wikia::setProps unfortunately fails silently so if we get to this point
+		 * we can set the response's status to true anyway...
+		 */
+		$this->response->setVal( 'status', true );
+
+		Wikia::setProps( $pageId, [
+			self::TEMPLATE_INFOBOX_PROP => 0,
```

Comment:
I think we're using two different sets of props, @adamkarminski -- I'd suggest following the one set out in #7660 -- can we get that one reviewed and merged and use TemplateClassificationController here?